src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/PaymentServiceRequestHandler.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.payment;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import feign.FeignException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.genapplication.GAPbaDetails;
+import uk.gov.hmcts.reform.civil.service.PaymentsService;
+import uk.gov.hmcts.reform.civil.service.Time;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackParams.Params.BEARER_TOKEN;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.MAKE_PAYMENT_SERVICE_REQ_GASPEC;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class PaymentServiceRequestHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(MAKE_PAYMENT_SERVICE_REQ_GASPEC);
+    private static final String ERROR_MESSAGE = "Technical error occurred";
+    private static final String TASK_ID = "GeneralApplicationMakePayment";
```

Comment:
This task id is bit confusing and not reflecting what the task is meant to do... would recommend renaming task id

Reply:
Modified the name of Task_ID to GeneralApplicationPaymentServiceReq